tests/test_cache_reporter.py

```
@@ -0,0 +1,398 @@
+"""
+Testes para o CacheReporter - Sistema de relatórios e monitoramento de cache.
+"""
+
+import pytest
+import tempfile
+import time
+import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)